frontend/src/pages/login.tsx

```diff
@@ -202,11 +230,13 @@ const Login = () => {
                   registerViaEmailAndPassword({
                     email: email,
                     password: password,
-                    displayName: fullName,
+                    passwordConfirmation: confirmPassword,
+                    firstName: firstName,
```

Comment:
what if we make the display name "firstName + " " + lastName" so that we don't have to concat first and last name each time (or have it stored in the db accordingly)

Reply:
The concatenation logic is done in userLogin.ts after the format of both the first name and last name are verified.

Reply:
got it, so we don't need to worry abt always doing concat?

Reply:
I don't think so.